### PR TITLE
Creation of the Neural Net Builder

### PR DESCRIPTION
**Pull request by RamBITS-AI**
Creating the Neural Net Builder.

### CODE REVIEW

File: templates/index.gohtml [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <!-- Include Semantic UI CSS -->
     <link rel="stylesheet" type="text/css" href="/static/semantic.css">
     <link rel="stylesheet" href="/static/codemirror.min.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.5.0/components/icon.min.css" integrity="sha512-rTyypI20S663Wq9zrzMSOP1MNPHaYX7+ug5OZ/DTqCDLwRdErCo2W30Hdme3aUzJSvAUap3SmBk0r5j0vRxyGw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (spookieoli) [inline templates/index.gohtml:9]: we have semantic under static/ please add the file to the static folder

**Review by spookieoli — CHANGES_REQUESTED** [review]
will add some response from neural net, please add the semanticui file to the static folder so we can load it from a trusted source - even if we are on a isolated on premise location

File: templates/index.gohtml (hunk 1 above)
Comment (spookieoli) [reply templates/index.gohtml:9]: added the file

**Review by spookieoli — APPROVED** [review]

**Comment by spookieoli** [comment]
Added changes to js to get updates from Trainingprogress every second, this is ready for testing

File: templates/neuralnetbuilder.gohtml [hunk 2]
```diff
@@ -0,0 +1,341 @@
+<!DOCTYPE html>
+<html lang="en">
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Neural Net Builler</title>
+    <link rel="stylesheet" type="text/css" href="/static/semantic.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.5.0/components/icon.min.css" integrity="sha512-rTyypI20S663Wq9zrzMSOP1MNPHaYX7+ug5OZ/DTqCDLwRdErCo2W30Hdme3aUzJSvAUap3SmBk0r5j0vRxyGw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (RamBITS-AI) [inline templates/neuralnetbuilder.gohtml:9]: There is one here as well.
```suggestion
    <link rel="stylesheet" href="/static/icon.min.css"/>
```

**Review by spookieoli — APPROVED** [review]
Great Work!

**Pull request merged**